contrib/externs/sinon-1.17.js

```diff
@@ -854,3 +854,128 @@ SinonFakeXmlHttpRequest.prototype.respond = function(status, headers, body) {};
  * @return {SinonClock}
  */
 sinon.useFakeTimers = function(params) {};
+
+/**
+ * Assertions
+ * @see http:/sinonjs.org/releases/1.17.7/assertions
+ */
+sinon.assert = {};
+
+/** @type {string} */
+sinon.assert.failException;
+
+/** @param {string} message */
+sinon.assert.fail = function(message) {};
+
+/** @param {*} assertion */
+sinon.assert.pass = function(assertion) {};
+
+/** @param {!SinonSpy} spy */
+sinon.assert.notCalled = function(spy) {};
+
+/** @param {!SinonSpy} spy */
+sinon.assert.called = function(spy) {};
+
+/** @param {!SinonSpy} spy */
+sinon.assert.calledOnce = function(spy) {};
+
+/** @param {!SinonSpy} spy */
+sinon.assert.calledTwice = function(spy) {};
+
+/** @param {!SinonSpy} spy */
+sinon.assert.calledThrice = function(spy) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {number} count
+ */
+sinon.assert.callCount = function(spy, count) {};
+
+/** @param {...SinonSpy} spies */
+sinon.assert.callOrder = function(...spies) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {*} obj
+ */
+sinon.assert.calledOn = function(spy, obj) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {*} obj
+ */
+sinon.assert.alwaysCalledOn = function(spy, obj) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {...*} args
+ */
+sinon.assert.calledWith = function(spy, args) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {...*} args
+ */
+sinon.assert.alwaysCalledWith = function(spy, args) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {...*} args
+ */
+sinon.assert.neverCalledWith = function(spy, args) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {...*} args
+ */
+sinon.assert.calledWithExactly = function(spy, args) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {...*} args
+ */
+sinon.assert.alwaysCalledWithExactly = function(spy, args) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {...*} args
+ */
+sinon.assert.calledWithMatch = function(spy, args) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {...*} args
+ */
+sinon.assert.alwaysCalledWithMatch = function(spy, args) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {...*} args
+ */
+sinon.assert.neverCalledWithMatch = function(spy, args) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {*=} exception
+ */
+sinon.assert.threw = function(spy, exception) {};
+
+/**
+ * @param {!SinonSpy} spy
+ * @param {*=} exception
+ *
+ * */
+sinon.assert.alwaysThrew = function(spy, exception) {};
+
+/**
+ * @typedef {Object} SinonExposeOptions
+ * @property {string=} prefix
```

Comment:
@property is not a valid annotation in closure-compiler JSDoc.

Ths should be
```javascript
/**
 * @typedef {{
 *     prefix: string|undefined,
 *     includeFail: boolean|undefined
 * }}
 */
var SinonExposeOptions;
```

Reply:
done